sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -184,7 +184,7 @@ case class FileSourceScanExec(
     "Batched" -> supportsBatch.toString,
     "PartitionFilters" -> partitionFilters.mkString("[", ", ", "]"),
     "PushedFilters" -> dataFilters.mkString("[", ", ", "]"),
-    "InputPaths" -> relation.location.paths.mkString(", "))
+    "RootPaths" -> relation.location.rootPaths.mkString(", "))
```

Comment:
I'm not quite sure what you're asking for here. If I replace `"RootPaths"` with an entry

    "Location" -> relation.location

and implement appropriate `toString` methods for the `relation.location` classes, does that satisfy your concern?